src/uct/api/uct.h

```diff
@@ -134,14 +134,17 @@ BEGIN_C_DECLS
  * @ref uct_ep_create
  *      Connect to the client by creating an endpoint if the request is accepted.
  *      The server creates a new endpoint for every connection request that it accepts.
- * @ref uct_cm_ep_priv_data_pack_callback_t
- *      This callback is invoked by the UCT transport to fill auxiliary data in
- *      the connection acknowledgement or reject notification back to the client.
- *      Send the client a connection acknowledgement or reject notification.
- *      Wait for an acknowledgment from the client, indicating that it is connected.
  * @ref uct_cm_ep_server_conn_notify_callback_t
  *      This callback is invoked by the UCT transport to handle the connection
  *      notification from the client.
+ * @note If private data has to be sent to client side and was not provided at
+ *       @ref uct_ep_create step by some reason, it can be done with
+ *       @ref uct_cm_ep_priv_data_pack_callback_t, then it should be set in
+ *       @ref uct_ep_params_t. This callback is invoked by the UCT transport to
+ *       fill auxiliary data in the connection acknowledgement or reject
+ *       notification back to the client. Send the client a connection
+ *       acknowledgement or reject notification. Wait for an acknowledgment from
+ *       the client, indicating that it is connected.
```

Comment:
note
The private data which the server should send to the client can be either provided directly to uct_ep_create, or filled by uct_cm_ep_priv_data_pack_callback_t provided to uct_ep_create.
note
In order to reject a connection request, can either call XYZ or return XYZ from uct_cm_ep_priv_data_pack_callback_t